domain/src/main/kotlin/no/nav/su/se/bakover/domain/LukketSøknadBrevinnhold.kt

```diff
@@ -32,21 +31,19 @@ sealed class LukketSøknadBrevinnhold {
         override val bruksenhet: String?,
         override val postnummer: String?,
         override val poststed: String,
-        override val typeLukking: String = Søknad.TypeLukking.Trukket.value
     ) : LukketSøknadBrevinnhold() {
 
         companion object {
             fun lagTrukketSøknadBrevinnhold(
                 person: Person,
                 søknad: Søknad,
-                lukketSøknadBody: Søknad.LukketSøknadBody
+                lukketSøknad: Søknad.Lukket.Trukket
             ): TrukketSøknadBrevinnhold {
                 return TrukketSøknadBrevinnhold(
-                    dato = LocalDate.now().format(DateTimeFormatter.ofPattern("dd.MM.yyyy")).toString(),
+                    dato = LocalDate.now().format(DateTimeFormatter.ofPattern("dd.MM.yyyy")),
```

Comment:
Vi kunne laget en extensionfunction for brev: `fun LocalDate.brevFormat() = this.format(DateTimeFormatter.ofPattern("dd.MM.yyyy") `

Reply:
todo: Finn bedre navn enn det =)

Reply:
👍